### DISCUSSION TITLE
Add device information into text for alexa

### DISCUSSION THREAD

**Discussion by eduspargoli** [keatontaylor/alexa-actions#249] [Q&A] [answered]
Hello Folks, I'm wondering if I can add a device information variable in the text that alexa speaks, for example:

"Text": " Welcome Eduardo. The current temperature is {{ states('sensor.ar_cond_room_temperature') }} would you like me to turn on the AC?"

do you know if this is possible?

**Comment by DEADSEC-SECURITY** [comment, marked as answer]
Just do it like this: `"Welcome Eduardo. The current temperature is" + states('sensor.ar_cond_room_temperature') + "would you like me to turn on the AC?"`

That works when I tested but let me know if it doesn't on your side. i would recommend you use the developer tools template tester from home assistant for testing stuff like this.

**Reply by eduspargoli** [reply]
Not luck on my side, but it's my bad... I'm doing it on node red, all my automations are in there. I'm getting an error message on the debug node: "JSONataError: Attempted to invoke a non-function"

that's how I did:

{	   "text":"Vejo que a temperatura está em" + states('sensor.ar_cond_room_temperature') + "Você quer que eu ligue o ar condicionado?",	   "event_id":"TEST",	   "alexa_device":"media_player.echo_dot_bedroom",	   "suppress_confirmation":"false"	}

**Reply by DEADSEC-SECURITY** [reply]
If your doing in node red then it wont work. you have to sue javascript and a node to the entity state. That only works for home assistant automations
